plugins/inventory/lxd.py

```diff
@@ -926,6 +932,8 @@ def group_type(group_name):
                 self.build_inventory_groups_vlanid(group_name)
             elif self.groupby[group_name].get('type') == 'type':
                 self.build_inventory_groups_type(group_name)
+            elif self.groupby[group_name].get('type') == 'project':
```

Comment:
You need to update the documentation, it doesn't mention `project` so far. Also add an example please.

Reply:
done.